.github/workflows/predict-conflicts.yml

```
@@ -23,10 +23,38 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: check for potential conflicts
+        id: check_conflicts
         uses: PastaPastaPasta/potential-conflicts-checker-action@v0.1.10
         with:
           ghToken: "${{ secrets.GITHUB_TOKEN }}"
       - name: Checkout
         uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)